### PR TITLE
BICAWS-1875 Use native types 

### PR DESCRIPTION
**Pull request by C-gyorfi**
### Changes:
- Update @moj-bichard7-developers/bichard7-next-data package
- Use boolean types for `RecordableOnPNCindicator` and `NotifiableToHOindicator` in AHO schema. These fields indicating boolean values but previously used strings to represent true or false
- Update snapshots

### CODE REVIEW

File: src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const enrichOffence = (offence: Offence): Offence => {
 
   if (offenceIgnored) {
     offence.OffenceCategory = "B7"
-    offence.RecordableOnPNCindicator = "N" // other possible options are "D" (don't know), "NA" (not applicable)
+    offence.RecordableOnPNCindicator = false // other possible options are "D" (don't know), "NA" (not applicable)
```
Comment (tomvaughan77) [inline src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts:18]: If `RecordableOnPNCindicator` has other possible options, does it belong as an enum instead?

File: tests/exceptions/HO100236.test.ts [hunk 2]
```diff
@@ -24,10 +24,10 @@ describe("HO100233", () => {
       cjsCode: "MC8080524",
       offenceCategory: "CB",
       offenceTitle: "Application to reopen case",
-      recordableOnPnc: "N",
+      recordableOnPnc: false,
       description: "blah",
       homeOfficeClassification: "123/45",
-      notifiableToHo: "Y",
+      notifiableToHo: false,
```
Comment (tomvaughan77) [inline tests/exceptions/HO100236.test.ts:30]: Should be `true`

Comment (C-gyorfi) [reply tests/exceptions/HO100236.test.ts:30]: Ooops, that is not looking good

Comment (C-gyorfi) [reply tests/exceptions/HO100236.test.ts:30]: I'll have a look

Comment (C-gyorfi) [reply tests/exceptions/HO100236.test.ts]: Good catch, updated, although I found it was irrelevant for the test

File: src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts (hunk 1 above)
Comment (C-gyorfi) [reply src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts:18]: Looks like we only use it as a boolean(Y or N) in the core repo, not sure about old Bichard 😕

Comment (C-gyorfi) [reply src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts]: Removed comment in [d043f17](https://github.com/ministryofjustice/bichard7-next-core/pull/72/commits/d043f17a672fa22ed44b4cdd4840e28ceac0bb49)

**Review by tomvaughan77 — APPROVED** [review]

**Review by tomvaughan77 — APPROVED** [review]

**Pull request merged**